lib/matplotlib/patches.py

```diff
@@ -3505,11 +3513,13 @@ class CurveFilledB(_Curve):
 
         def __init__(self, head_length=.4, head_width=.2):
             """
-            *head_length*
-              length of the arrow head
+            Parameters
+            ----------
+            head_length : float, optional, default : .4
```

Comment:
@QuLogic 

You are correct. From the numpydoc spec:

> If it is not necessary to specify a keyword argument, use optional:
> 
> `x : int, optional`
> Optional keyword parameters have default values, which are displayed as part of the function signature. They can also be detailed in the description:
> 
```
Description of parameter `x` (the default is -1, which implies summation
over all axes).
If it is not necessary to specify a keyword argument, use optional:
```
> 
> `x : int, optional`
> Optional keyword parameters have default values, which are displayed as part of the function signature. They can also be detailed in the description:
>  
> `Description of parameter `x` (the default is -1, which implies summation over all axes).`

(though I wish the meaning of "optional" in this context was a little more precise.